recipes-kernel/linux/linux-milkv-duo-dev.bb

```diff
@@ -0,0 +1,47 @@
+SUMMARY = "Milk-V Duo dev kernel recipe"
+
+inherit kernel
+require recipes-kernel/linux/linux-yocto.inc
+
+DEPENDS = "u-boot-mkimage-native dtc-native"
+
+BRANCH ?= "main"
+FORK ?= "xyq1113723547"
+
+#  git://git.yoctoproject.org/yocto-kernel-cache;type=kmeta;name=meta;branch=master;destsuffix=${KMETA} \
```

Comment:
I think it will be better to use 5.10 branch of meta tree as well. see https://git.yoctoproject.org/yocto-kernel-cache/log/?h=yocto-5.10

Reply:
Make sense.

Reply:
> I think it will be better to use 5.10 branch of meta tree as well. see https://git.yoctoproject.org/yocto-kernel-cache/log/?h=yocto-5.10

em,  my second thought, the above comment left there only makes people confused, the milkv duo doesn't work with yocto-kernel-cache's yocto-5.10 branch, right? as it requires quite heavy extra patches.. so we should have that comment removed